spec/schema/schema_dumper_spec.rb

```diff
@@ -0,0 +1,69 @@
+describe "SchemaDumper" do
+  SCHEMA_RB_FILE  = File.expand_path("../dummy/db/schema.rb", __dir__)
+  TABLE_LIST_FILE = File.expand_path("../support/table_list.txt", __dir__)
+
+  let(:combined_metric_list) { metric_rollup_list + metric_list }
+  let(:metric_list)          { (0..23).to_a.map { |num| "metrics_#{'%<num>02d' % { :num => num }}" } }
```

Comment:
Might be cleaner / easier to read?

```suggestion
  let(:metric_list)          { (0..23).to_a.map { |num| "metrics_#{num.to_s.rjust(2, '0')}" } }
```

Reply:
Well... it was smaller than that prior to `rubocop` requiring the formatted names...

```
'%02d' % num

# v.s.

.to_s.rjust(2, '0')
```

Anyway, sure, I will change

Reply:
For someone that small, I'd be fine without the formatted names.  I think rubocop is wrong 😆

Reply:
Yeah, but I am just not going to bother, so I will go with your suggestion.  I just:

a) Forgot how to do it with `rjust`, and didn't look it up
b) Forgot `rubocop` had this rule...